mptcp-pm/examples/dump_mptcp.rs

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+
+use futures::stream::TryStreamExt;
+
+// TODO: convert to a test
```

Comment:
Is this "TODO" planned for this PR or for a follow-up patch?

Reply:
It is unclear github CI(Ubuntu) support mptcp or not yet. Let me give it a try.

Reply:
Integration test case added and enabled in github CI.